apps/element-storybook/src/layouts/form/Form.stories.tsx

```diff
@@ -0,0 +1,583 @@
+// Each exported component in the package should have its own stories file
+import React, { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+import { Radio, RadioGroup, FormControl, InputAdornment } from '@mui/material';
+import { TextField } from '@availity/mui-textfield';
+import { Autocomplete } from '@availity/mui-autocomplete';
+import { Datepicker } from '@availity/mui-datepicker';
+import { FormControlLabel, FormLabel } from '@availity/mui-form-utils';
+import { Button } from '@availity/mui-button';
+import { PageHeader } from '@availity/mui-page-header';
+import { Paper } from '@availity/mui-paper';
+import { Typography } from '@availity/mui-typography';
+import { Grid, Container } from '@availity/mui-layout';
+import { SearchIcon } from '@availity/mui-icon';
+import { Divider } from '@availity/mui-divider';
+import dayjs from 'dayjs';
+
+const meta: Meta = {
+  title: 'Layouts/Form',
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+const SectionText = ({ marginBottom }: { marginBottom: string }) => (
+  <Typography paragraph sx={{ marginBottom }}>
+    This is introduction or helper text that helps the user understand the data that needs to be entered or selections
+    that need to be made in this section.
+  </Typography>
+);
+
+const AsteriskMessage = ({ marginBottom }: { marginBottom: string }) => (
+  <Typography paragraph sx={{ marginBottom }}>
+    <Typography component="span" color="error.main" variant="inherit">
+      *
+    </Typography>{' '}
+    is a required field.
+  </Typography>
+);
+
+type SimpleFormInputs = {
+  text: string;
+  dropdown: string;
+};
+
+const dropdownOptions = ['Bulbasaur', 'Squirtle', 'Charmander'];
+
+export const _SimpleForm: StoryObj = {
+  render: () => {
+    const {
+      register,
+      handleSubmit,
+      formState: { errors },
+      getValues,
+      control,
+      reset,
+    } = useForm<SimpleFormInputs>();
+
+    const [submitted, setSubmitted] = useState(false);
+
+    const onSubmit: SubmitHandler<SimpleFormInputs> = (data) => {
+      console.log(data);
+      setSubmitted(true);
+    };
+
+    return (
+      <Container>
+        <Paper sx={{ padding: '1.5rem' }}>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <Typography component="h1" variant="h2" sx={{ marginBottom: '.5rem' }}>
+              Simple Form
+            </Typography>
+            <SectionText marginBottom="1.5rem" />
+            <AsteriskMessage marginBottom="1.5rem" />
+            <Grid container marginBottom="1.5rem">
```

Comment:
iirc the entire grid should be in one container, then you can set the `columnSpacing={3}` and `rowSpacing='1'` once.